pyrasterframes/src/main/python/pyrasterframes/__init__.py

```diff
@@ -146,6 +149,16 @@ def temp_name():
     if band_indexes is None:
         band_indexes = [0]
 
+    if spatial_index_partitions is False:
```

Comment:
I propose the below which covers some additional cases e.g. user passing in a literal `int` 0.

Note that when `a is None` then a logical test on `a` returns False. 

```suggestion
    if spatial_index_partitions:
        if int(spatial_index_partitions) < 1:
            spatial_index_partitions = '-1'
       
        options.update({"spatial_index_partitions": str(int(spatial_index_partitions))})
```